mysite/code/HomePage.php

```diff
@@ -1,9 +1,8 @@
 <?php
 
-class HomePage extends BaseHomePage {
-	private static $description = 'Entry to the site. Has carousel, featured panels, quicklinks and news summaries';
```

Comment:
Do you want to keep this description?

Reply:
`cwp/agency-extensions` isn't installed by default, so the carousel is out.
I didn't think it was necessary any longer, and I'm not sure where/how it is used - and it was removed when I thought this was to be a more 'bare bones' recipe than it is.

Would you like it added back?

Reply:
True, all good